ga4gh/backend.py

```diff
@@ -404,6 +405,89 @@ def _topLevelObjectGenerator(self, request, numObjects, getByIndexMethod):
                 nextPageToken = str(currentIndex)
             yield object_.toProtocolElement(), nextPageToken
 
+    def _filterReadGroupsByBioSampleId(self, request, readGroupSet):
+        newRg = []
+        for rg in readGroupSet.readGroups:
+            print(rg.bioSampleId, request.bioSampleId)
+            if request.bioSampleId == rg.bioSampleId:
+                newRg.append(rg)
+        readGroupSet.readGroups = newRg
+        return readGroupSet
+
+    def _readGroupSetObjectGenerator(
```

Comment:
Thanks added to this PR https://github.com/ga4gh/schemas/pull/579/files#diff-0b6f468d7591bb264e8eb143b119e890R158